Mint/README.md

```diff
@@ -1,35 +1,149 @@
 # Linux Mint
 
 ## It's History
-``-> Add distro history + What it is based on? + Why others made that distro?``
 
-## Distro images
-``-> Add distro images``
+Linux mint is a distribution based on Ubuntu (which in turn is based on Debian).
+
+The linux mint project was strated by a software developer called Clément Lefèbvre (now known as Clem). Clem at the time was writing reviews and tuotorials for various linux websites and decided to start his own linux blog which he called "linux mint" (yes that where the naming came from).
```

Comment:
- Typo: **strated -> started**
- Typo: **tuotorials -> tutorials**
- Grammatical fix: (yes **that's** where the **name** came from).

Reply:
Still need to fix:
- Grammatical fix: (yes **that's** where the **name** came from).